src/test/java/Drivers/ChromeDriverManager.java

```diff
@@ -0,0 +1,22 @@
+package Drivers;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class ChromeDriverManager {
+    public WebDriver getDriver() {
+        System.setProperty("webdriver.chrome.driver", "src/test/java/resources/chromedriver.exe");
```

Comment:
Hardcoded Path. We should consider some configuration file with appropriate paths or environment variables.

Reply:
@fnfldx Fixed now.